Dockerfile

```diff
@@ -0,0 +1,74 @@
+# Install PHP libraries
+FROM composer:2.0 as composer
+
+WORKDIR /usr/local/src/
+
+COPY composer.lock /usr/local/src/
+COPY composer.json /usr/local/src/
+
+RUN composer install --ignore-platform-reqs --optimize-autoloader \
+    --no-plugins --no-scripts --prefer-dist
+
+# Prepare generic compiler
+FROM php:8.0.18-cli-alpine3.15 as compile
+
+ENV PHP_SWOOLE_VERSION=v4.8.10
+
+RUN \
+  apk add --no-cache --virtual .deps \
+  make \
+  automake \
+  autoconf \
+  gcc \
+  g++ \
+  git \
+  openssl-dev
+  
+RUN docker-php-ext-install sockets
+
+# Compile Swoole
+FROM compile AS swoole
+
+RUN \
+  git clone --depth 1 --branch $PHP_SWOOLE_VERSION https://github.com/swoole/swoole-src.git && \
+  cd swoole-src && \
+  phpize && \
+  ./configure --enable-sockets --enable-http2 --enable-openssl && \
+  make && make install && \
+  cd ..
+
+# Proxy
+FROM php:8.0.18-cli-alpine3.15 as final
+
+LABEL maintainer="team@appwrite.io"
+
+RUN \
+  apk update \
+  && apk add --no-cache --virtual .deps \
+  make \
+  automake \
+  autoconf \
+  gcc \
+  g++ \
+  curl-dev \
+  && apk add --no-cache \
+  libstdc++ \
+  && docker-php-ext-install sockets \
+  && apk del .deps \
+  && rm -rf /var/cache/apk/*
+
+WORKDIR /usr/local/
+
+# Source code
+COPY ./app /usr/local/app
+COPY ./src /usr/local/src
+
+# Extensions and libraries
+COPY --from=composer /usr/local/src/vendor /usr/local/vendor
+COPY --from=swoole /usr/local/lib/php/extensions/no-debug-non-zts-20200930/swoole.so /usr/local/lib/php/extensions/no-debug-non-zts-20200930/yasd.so* /usr/local/lib/php/extensions/no-debug-non-zts-20200930/
```

Comment:
We need to double check yasd can be installed without damaging performance in production

Reply:
Removing yasd. I removed swoole debugger anyway.

Or is swoole debugger important? Should I keep it as part of Open Runtimes Proxy image?

Reply:
Not crucial and can be helpful only during development

Reply:
Ill keep it off until we find a need for it. We can always just copy it from Appwrite. Open source 🌈